pkg/httputil/httputil.go

```diff
@@ -45,6 +47,29 @@ func GetWithUserAgent(url string) (*http.Response, error) {
 	return resp, nil
 }
 
+// PostWithUserAgent performs an HTTP POST with the proper Clair User-Agent.
+func PostWithUserAgent(url string, data interface{}) (*http.Response, error) {
+	client := &http.Client{}
+	postData, err := json.Marshal(data)
+	if err != nil {
+		return nil, err
+	}
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer(postData))
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Set("Content-Type", "application/json")
```

Comment:
If this is supposed to be more generic perhaps I wouldn't hardcode content type?